python/taichi/lang/impl.py

```diff
@@ -285,8 +285,14 @@ def __repr__(self):
 root = Root()
 
 
-@python_scope
+#@deprecated('ti.var', 'ti.field')
```

Comment:
Are these `@deprecated` commented out on purpose?

Reply:
Yes, we don't want to throw out tons of warnings yet, before it's fully deprecated from tests and examples. The `#@` is just a reminder, this is **to be deprecated in the future**.